### PR TITLE
Added Home Page Slider

### LINKED ISSUE #193: Create Slider for Home Page
## Description
As a contributor,
I need create a Home Page,
so that users can get easy access to the information about Systers.

## Mocks
screen shot 2018-05-31 at 10 49 15 pm

### Update [Required]
- [x] home.component.html
- [x] home.component.ts

## Definition of Done
- [x] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
6 hours 

@ancyp @Janiceilene @divyanshu-rawat Can you please assign me this issue to work upon. :)

### PR DESCRIPTION
**Pull request by BaaniLeen**
### Description
Added a slider on the Home Page using Angular carousel, and ngFor to ensure code modularity.

In order to add/remove/change an image, only the image path and description need to be added in the slides array in home.component.ts file, hereon.

Fixes #193 

### Type of Change:
- Code
- User Interface
  
**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
This is the demo of how it looks.
https://drive.google.com/file/d/1agRLN6y3VGXOXlu5wcZnk4jzGrEUGUb_/view?usp=sharing.

Mobile Responsiveness:
https://drive.google.com/file/d/1vS29F8yXYNdNDx7ir9gebBfA_Ro0O6NQ/view?usp=sharing

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
  

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by ancyp** [comment]
mention mobile testing!
Footer goes the full width of the page =>was Issue for this created?

**Comment by BaaniLeen** [comment]
@ancyp Sure :)
I was waiting for the confirmation regarding which all tab headings need to be added in the header and then combine both the points and make a new issue. No worries. I will do it right away :)

**Comment by BaaniLeen** [comment]
@ancyp Here is the link for the demo regarding mobile responsiveness of the Home Page Slider:
https://drive.google.com/file/d/1vS29F8yXYNdNDx7ir9gebBfA_Ro0O6NQ/view?usp=sharing :)

**Comment by ancyp** [comment]
@BaaniLeen need not give demo link. I meant just a checklist option for saying - it has been tested on mobile. :)

**Comment by ancyp** [comment]
screen shot 2018-06-01 at 5 24 09 pm

The whole Image should fit within the screen (currently have to scroll). May be we can fit the image fully on the screen without left and right margins. @Janiceilene what do you think?

**Comment by Janiceilene** [comment]
I agree with the photo going the full width and definitely should be short enough that there's no need to scroll.  Didn't we discuss removing those types of images for mobile versions, @BaaniLeen ?

**Comment by divyanshu-rawat** [comment]
This is how slider is looking in web page & Mobile device.
Kidnly fix it.

Web UI.
screen shot 2018-06-03 at 11 36 10 pm

Mobile UI
screen shot 2018-06-03 at 11 36 27 pm

**Comment by divyanshu-rawat** [comment]
Situation gets worse when navigation-bar is opened.
screen shot 2018-06-03 at 11 38 48 pm

**Comment by BaaniLeen** [comment]
@divyanshu-rawat I have already created a new issue for the responsiveness of the Navbar and the footer and am trying to fix that. 
Can you please comment out the <app-footer> from the app.component.html tag and then look at the mobile responsiveness of the Home Page slider alone for this PR, as shown in the video attached?
Thanks :)

**Comment by divyanshu-rawat** [comment]
@BaaniLeen I wil check and let you know 👍

File: src/app/home/home.component.html [hunk 1]
```diff
@@ -1 +1,12 @@
-<h1> HOME PAGE</h1>
+<div class=" container box logos mx-auto d-block mt-5 mb-5">
```
Comment (ancyp) [inline src/app/home/home.component.html]: extra space

**Comment by BaaniLeen** [comment]
@ancyp Can you please approve and merge this PR. I have made the suggested changes. :)

**Pull request merged**